run-clean.sh

```diff
@@ -0,0 +1,2 @@
+docker system prune -a -f
+docker compose up -d && docker-compose exec -it apache bash -c "composer install && chown -R www-data:www-data /var/www/html/vendor"
```

Comment:
```suggestion
docker compose down && docker compose up -d --build && docker-compose exec -it apache bash -c "composer install && chown -R www-data:www-data /var/www/html/vendor"
```
This will bring down the containers and rebuild the images without touching images/containers not involved with this project